src/components/AccordionList.tsx

```diff
@@ -0,0 +1,32 @@
+import { useState } from "react";
+import { useTestIds } from "..";
+import { Accordion, AccordionProps, AccordionSize } from "./Accordion";
+
+interface AccordionListProps {
+  accordions: AccordionProps[];
+  /** Allows multiple accordions to be expanded simultaneously (enabled by default) */
+  allowMultipleExpanded?: boolean;
+  size?: AccordionSize;
+}
+
+export function AccordionList({ accordions, size, allowMultipleExpanded = true }: AccordionListProps) {
+  const [expandedIndex, setExpandedIndex] = useState<number>();
+  const tid = useTestIds({}, "accordionList");
```

Comment:
**nit:** Could you update to pass `props` into `useTestids` (`useTestIds(props, "accordionList")`). Typically in Beam where we are making generic components, it makes sense to use the component's `props` as the first param of `useTestIds`. This allows the caller to define their own custom testid for the component if they wish. 
In `internal-frontend` we don't do that as much because our components aren't as generic and it doesn't make sense to allow for overriding the default test id.